libs/transactions/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@vegaprotocol/transactions",
+  "version": "0.1.0",
```

Comment:
**suggestion (non-blocking)**: This would be clearer if it matched the core version the transactions were appropriate for (so I don't need to know that `0.1.0` is for `0.67.3`)